tests/conftest.py

```diff
@@ -2728,6 +2728,8 @@ def ceph_toolbox(request):
     deploy = config.RUN["cli_params"]["deploy"]
     teardown = config.RUN["cli_params"].get("teardown")
     skip_ocs = config.ENV_DATA["skip_ocs_deployment"]
+    if config.ENV_DATA["platform"].lower() == "openshiftdedicated":
+        skip_ocs = False
```

Comment:
Can you please add some comment why we do that.  As we discussed offline I would like to see if we will run post deployment verification which we have in our code but I see that the OCS deployment is done behind the scenes for us and that we will probably not do the changes we do for deployment as well but as we discussed after the deployment we are running some extra code for monitoring and so on which in this case will not be executed.

Reply:
Thanks, Added comments in the code base.

Reply:
Ah, this case I would rather change the code of the condition to proposed in the other comment